server/graphql/v2/object/Host.js

```diff
@@ -117,7 +121,15 @@ export const Host = new GraphQLObjectType({
           },
         },
         async resolve(host, args) {
-          const metrics = await host.getHostMetrics(args.dateFrom || args.from, args.dateTo || args.to);
+          let collectiveIds;
+          if (args.account) {
+            const collectives = await fetchAccountsWithReferences(args.account, {
+              throwIfMissing: true,
```

Comment:
Maybe we don't have to throw an error here?

Reply:
Thanks, makes sense to not throw by errors here if one collective id/slug is missing. 🤔 I've corrected this. 👍🏽